parsing/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -2653,6 +2653,8 @@ SqlNode SqlStmt() :
         stmt = SqlMerge()
     |
         stmt = SqlProcedureCall()
+    |
+        stmt = SqlHelp()
```

Comment:
nit: Order alphabetically.

Reply:
Should I reorder the rest of the calls here since they are all out of order?